CIP-0069/README.md

```diff
@@ -0,0 +1,87 @@
+---
+CIP: 0069
+Title: Script Signature Unification 
+Authors: Maksymilian 'zygomeb' Brodowicz <zygomeb@gmail.com>
+Status: Draft
+Type: Standards Track
+Created: 2022-08-23
+License: CC-BY-4.0
+---
+
+## Simple Summary / Abstract
+
+This CIP unifies the arguments given to all types of Plutus scripts currently available (spending, certifying, rewarding, minting) by removing the argument of a datum.
+
+For a while now every builder, myself included have struggled with the mutual dependency issue (two validators that need to know each other's hash) when designing dapps and it is widely considered a substantial barrier to safe protocols and one that limits our design space considerably. 
+
+The exact change would be to have every validator take as argument the redeemer and then the script context. Datums, only relevant to locking validators would be able to be provided by either looking them up in the ScriptContext or by extending the `Spending` constructor of `TxInfo` to carry `(TxOutRef, Datum)`. 
+
+If this CIP were to be accepted and implemented, we'd be able to resolve a mutual dependency by merging two scripts into one, that can act as both say minting and spending depending on the purpose given. 
```

Comment:
The last sentence is unnecessary, or at least, needs reformulation. Write if from the perspective of the CIP being adopted. If it is rejected, it will be marked as such and if it is accepted, it should be read from that point of view.